test/regression-evm/Makefile

```diff
@@ -1,3 +1,4 @@
+DEF_KORE =
```

Comment:
Same question as above.

Reply:
`make build-test` is called for each test, and if I don't set `DEF_KORE` to be nothing, it's going to automatically search for a `.k` definition, which in this case doesn't exist.
I added the target `build-test` because Everett suggested it, but now I'm not so sure it's needed for what we are trying to do here.